client/modules/installation/components/tabs/installation.tsx

```diff
@@ -1,10 +1,35 @@
 "use client"
 
-import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs"
+import { useI18n } from "@/utils/services/internationalization/client"
 
-import { tabsData } from "../../data"
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs"
 
 const InstallationTabs = () => {
+  const t = useI18n()
+
+  const tabsData = [
+    {
+      title: "IOS",
+      value: "ios",
+      steps: [
+        t("pwa.ios.step_one"),
+        t("pwa.ios.step_two"),
+        t("pwa.ios.step_three"),
+        t("pwa.ios.step_four"),
+      ],
+    },
+    {
+      title: "Android",
+      value: "android",
+      steps: [
+        t("pwa.android.step_one"),
+        t("pwa.android.step_two"),
+        t("pwa.android.step_three"),
+        t("pwa.android.step_four"),
+      ],
+    },
+  ]
```

Comment:
The `tabsData` can be memoized using the `useMemo` hook to prevent recreating the array from scratch when the `InstallationTabs` rerenders.

```suggestion
  const tabsData = useMemo(
    () => [
      {
        title: "IOS",
        value: "ios",
        steps: [
          t("pwa.ios.step_one"),
          t("pwa.ios.step_two"),
          t("pwa.ios.step_three"),
          t("pwa.ios.step_four"),
        ],
      },
      {
        title: "Android",
        value: "android",
        steps: [
          t("pwa.android.step_one"),
          t("pwa.android.step_two"),
          t("pwa.android.step_three"),
          t("pwa.android.step_four"),
        ],
      },
    ],
    [t],
  )
```

Reply:
On this page, there is no JavaScript logic or other complicated re-rendering. I prefer to avoid using it until there are performance issues.